convert_old_alw_to_new_structure.py

```diff
@@ -0,0 +1,26 @@
+import csv
+#csv file from which the original data is read
+with open('hr.contract (8).csv') as csvfile1: 
+	#csv file to which new data is written
+	with open('csvfile.csv','w') as csvfile2:
```

Comment:
Again please do not hardcode here... if it can be supplied via command line, great if not please introduce variables at the top of the file; also same point on descriptive variable as it makes your code easier to read.  Also  line 3 and line 5 can be written as one:

```python
with open('a',) as a, open('b', 'w') as b:
    do_something()
```